src/molecules/Tabs.tsx

```diff
@@ -128,35 +94,29 @@ const Tabs = withRouter(({ match }) => {
           exact
           render={() => (
             <StyledP>
-              I'm description Lorem ipsum dolor sit amet consectetur adipisicing
-              elit. Quo sapiente tenetur cum, quis, quia veniam quod fugit
-              eligendi cupiditate vero, sequi asperiores cumque eos ipsum
-              pariatur alias dolore saepe unde. I'm description Lorem ipsum
-              dolor sit amet consectetur adipisicing elit. Quo sapiente tenetur
-              cum, quis, quia veniam quod fugit eligendi cupiditate vero, sequi
-              asperiores cumque eos ipsum pariatur alias dolore saepe unde.
+              <Description dataDescription={description} />
             </StyledP>
           )}
         />
         <Route
           path={`${match.url}/info`}
           render={() => (
             <StyledP>
-              <ProductTable data={dataInfo} />
+              <ProductTable data={shopAttributes} />
             </StyledP>
           )}
         />
         <Route
           path={`${match.url}/review`}
           render={() => (
             <StyledP>
```

Comment:
Why to wrap product reviews in a paragraph? I think you should remove the `StyledP` wrapper here.

Reply:
I need it for the lg screen, like a grid.
I do not need table be to the border page

Reply:
Please remove `StyledP` and add a new component like `Grid`, if you don't have one already. Describe the alignment there.

```tsx
const Grid = () => {
  return (
    <Box
      width="THE WIDTH YOU WANT"
      maxWidth="100%"
      marginHorizontal="auto"
      paddingHorizontal={1} // the horizontal spacing you want
    />
  )
}
```